src/components/templates/protected-area/passwordGeneratorModal.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect, useState } from "react"
+// import PasswordVisualization from "./passwordVisualization"
+import { generatePassword } from "../../../libs/passwordGenerator"
+import "./passwordGenerator.scss"
+import "./password.style.scss"
+
+//show the password casually generated with generatePassword func, user can choose lenght with slider
+const PasswGenerator = (props: any) => {
+  const [passwLength, setPasswLength] = useState(8)
+  const [isUpperCase, setUpperCase] = useState(false)
+  const [isNumber, setNumber] = useState(false)
+  const [isSpecial, setSpecial] = useState(false)
+  const [isAmbiguous, setAmbiguous] = useState(false)
+
+  const [PasswValue, setPasswValue] = useState(
+    generatePassword(passwLength, isUpperCase, isNumber, isSpecial, isAmbiguous)
+  )
+
+  const { closeModal } = props
+
+  function handlepasswLength(e: any) {
+    setPasswLength(parseInt(e.target.PasswValue))
```

Comment:
PassValue? This is taking down my browser hahaha

Reply:
PasswValue not PassValue ahahaah there's a w!

Reply:
Shouldn't be
```suggestion
    setPasswLength(parseInt(e.target.value))
```
?